flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/A320_Neo_CDU_FlightPlanPage.js

```diff
@@ -313,6 +313,7 @@ class CDUFlightPlanPage {
                 let timeColor = color;
 
                 // Altitude
+                const hasAltConstraint = wp.legAltitudeDescription > 0 && wp.legAltitudeDescription < 6;
```

Comment:
Why do we check for `wp.legAltitudeDescription < 6` here? It looks like this is the condition to display `legAltitude1` in the flight plan. In the case of `wp.legAltitudeDescription < 6`, the relevant constraint is stored in `legAltitude1` according to this: https://github.com/flybywiresim/a32nx/blob/master/src/fmgc/src/types/fstypes/FSEnums.ts#L47

Reply:
Yes, the data is not a constraint for precision approaches (descriptors >= 6 are used for those).